src/set/set.module.ts

```diff
@@ -1,20 +1,18 @@
-import { Module } from '@nestjs/common';
-import { MongooseModule } from '@nestjs/mongoose';
-import { SharedModule } from '../shared/shared.module';
-import { TaskSchema } from '../set/entities/task.entity';
-import { SetSchema } from './entities/set.entity';
-import { SetController } from './set.controller';
-import { SetService } from './set.service';
+import { Module } from '@nestjs/common'
+import { MongooseModule } from '@nestjs/mongoose'
+import { TaskSchema } from '../set/entities/task.entity'
+import { SetSchema } from './entities/set.entity'
+import { SetController } from './set.controller'
+import { SetService } from './set.service'
 
 @Module({
-  // TODO MC: Does forFeature from Mongoose only import or instance? This could cause bugs..
-  imports: [
-    SharedModule,
-    MongooseModule.forFeature([{ name: 'Set', schema: SetSchema }]), 
-    MongooseModule.forFeature([{ name: 'Task', schema: TaskSchema }])
-  ],
-  controllers: [SetController],
-  providers: [SetService],
-  exports: [SetService]
+    // TODO MC: Does forFeature from Mongoose only import or instance? This could cause bugs..
+    imports: [
+        MongooseModule.forFeature([{ name: 'Set', schema: SetSchema }]),
```

Comment:
use Set.name and User.name

Reply:
or SetSchema.name? idk but that thing that is not a direct string

Reply:
I will try thank you